src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23951.cs

```diff
@@ -0,0 +1,24 @@
+#if !MACCATALYST
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue23951 : _IssuesUITest
+	{
+		public Issue23951(TestDevice device) : base(device) { }
+
+		public override string Issue => "[Android] Frame disappears when assigning GradientStops to LinearGradientBrush inside this Frame";
+
+		[Test]
+		[Category(UITestCategories.Frame)]
+		public void FrameCornerRadiusShouldnotChange()
+		{
+			App.WaitForElement("Button");
+			App.Tap("Button");
+			VerifyScreenshot();
```

Comment:
Snapshots available in the latest build.
![image](https://github.com/user-attachments/assets/fb48fb0e-41c7-4834-b256-d91befb00eb0)

Reply:
@jsuarezruiz, the snapshot has been added.